src/javascripts/constants/constant.js

```diff
@@ -0,0 +1,15 @@
+export const MESSAGE = {
```

Comment:
File name should not be constant.js, it does not make sense because it's already a constant